pages/requests/new/[ware].js

```diff
@@ -97,28 +100,52 @@ const NewRequest = () => {
           uiSchema={dynamicForm.uiSchema}
           validator={validator}
         >
-          <div className='row'>
-            <div className='col'>
-              <ShippingDetails
-                billingCountry={requestForm.billing.country}
-                shippingCountry={requestForm.shipping.country}
-                updateRequestForm={updateRequestForm}
-              />
-            </div>
-            <div className='col'>
-              <AdditionalInfo updateRequestForm={updateRequestForm} defaultRequiredDate={oneWeekFromNow} />
-            </div>
-          </div>
-          <Button
-            addClass='my-4 ms-auto d-block btn btn-primary'
-            label='Initiate Request'
-            type='submit'
-            size='large'
+          <StandardRequestOptions
+            defaultRequiredDate={oneWeekFromNow}
+            requestForm={requestForm}
+            updateRequestForm={updateRequestForm}
           />
         </Form>
-      }
+      ) : (
+        <BsForm
+          onSubmit={handleSubmit}
+          id={`new-${id}-request-form`}
+          noValidate
+          validated={validated}
+        >
+          <BlankRequestForm updateRequestForm={updateRequestForm} />
```

Comment:
since we're rendering the blank request form here, we don't need the "pages/requests/new/index.js" file.